Swiftfin tvOS/Views/VideoPlayer/NativeVideoPlayer.swift

```diff
@@ -37,7 +37,6 @@ struct NativeVideoPlayer: View {
             if let _ = videoPlayerManager.currentViewModel {
                 playerView
             } else {
-//                VideoPlayer.LoadingView()
```

Comment:
Actually leave this, this is just left over from forever ago for when this needs to be properly implemented.

Reply:
Added a TODO instead of leaving commented out code to make the intention a little bit more explicit.